CHANGELOG.md

```diff
@@ -19,6 +19,9 @@
 * Stabilize Web Share API.
   [#3882](https://github.com/rustwasm/wasm-bindgen/pull/3882)
 
+* Stop using reflect in WebIDL dictionary setters, instead create a shim to allow the wasm_bindgen macro to implement the function in javascript which avoids passing the attribute name string over the WASM/JS boundary. This increases the size of the Web API bindings but should be more efficient. Also, importing getters/setters from JS now supports specifying the JS attribute name as a string, e.g. `#[wasm_bindgen(method, setter = "x-cdm-codecs")]`.
```

Comment:
```suggestion
* Generate JS bindings for WebIDL dictionary setters instead of using `Reflect`. This increases the size of the Web API bindings but should be more performant. Also, importing getters/setters from JS now supports specifying the JS attribute name as a string, e.g. `#[wasm_bindgen(method, setter = "x-cdm-codecs")]`.
```